ddtrace/contrib/aws_lambda/patch.py

```diff
@@ -12,7 +12,25 @@
 from ddtrace.internal.wrapping import wrap
 
 
-log = get_logger(__name__)
+class DDLambdaLogger:
+    """Uses `DDLogger` to log only on cold start invocations."""
+
+    def __init__(self):
+        from datadog_lambda.cold_start import is_cold_start
+
+        self.logger = get_logger(__name__)
+        self.is_cold_start = is_cold_start()
+
+    def exception(self, msg, *args, exc_info=True, **kwargs):
+        if self.is_cold_start:
+            self.logger.error(msg, *args, exc_info=exc_info, **kwargs)
+
+    def warning(self, msg, *args, **kwargs):
+        if self.is_cold_start:
+            self.logger.warning(msg, *args, **kwargs)
+
+
+log = DDLambdaLogger()
```

Comment:
I wonder how come this doesn't complain about not finding `datadog_lambda` module....